core/image/astc/astc.go

```diff
@@ -152,4 +157,81 @@ func init() {
 			return dst, nil
 		})
 	}
+
+	compressionSupportMap := []converterLayout{
+		{image.RGBA_U8_NORM, RGBA_4x4},
+		{image.RGBA_U8_NORM, RGBA_5x4},
+		{image.RGBA_U8_NORM, RGBA_5x5},
+		{image.RGBA_U8_NORM, RGBA_6x5},
+		{image.RGBA_U8_NORM, RGBA_6x6},
+		{image.RGBA_U8_NORM, RGBA_8x5},
+		{image.RGBA_U8_NORM, RGBA_8x6},
+		{image.RGBA_U8_NORM, RGBA_8x8},
+		{image.RGBA_U8_NORM, RGBA_10x5},
+		{image.RGBA_U8_NORM, RGBA_10x6},
+		{image.RGBA_U8_NORM, RGBA_10x8},
+		{image.RGBA_U8_NORM, RGBA_10x10},
+		{image.RGBA_U8_NORM, RGBA_12x10},
+		{image.RGBA_U8_NORM, RGBA_12x12},
+		{image.SRGBA_U8_NORM, SRGB8_ALPHA8_4x4},
+		{image.SRGBA_U8_NORM, SRGB8_ALPHA8_5x4},
+		{image.SRGBA_U8_NORM, SRGB8_ALPHA8_5x5},
+		{image.SRGBA_U8_NORM, SRGB8_ALPHA8_6x5},
+		{image.SRGBA_U8_NORM, SRGB8_ALPHA8_6x6},
+		{image.SRGBA_U8_NORM, SRGB8_ALPHA8_8x5},
+		{image.SRGBA_U8_NORM, SRGB8_ALPHA8_8x6},
+		{image.SRGBA_U8_NORM, SRGB8_ALPHA8_8x8},
+		{image.SRGBA_U8_NORM, SRGB8_ALPHA8_10x5},
+		{image.SRGBA_U8_NORM, SRGB8_ALPHA8_10x6},
+		{image.SRGBA_U8_NORM, SRGB8_ALPHA8_10x8},
+		{image.SRGBA_U8_NORM, SRGB8_ALPHA8_10x10},
+		{image.SRGBA_U8_NORM, SRGB8_ALPHA8_12x10},
+		{image.SRGBA_U8_NORM, SRGB8_ALPHA8_12x12},
+	}
+
+	for _, conversion := range compressionSupportMap {
+		// Intentional local copy
+		conv := conversion
+		image.RegisterConverter(conv.src, conv.dst, func(src []byte, width, height, depth int) ([]byte, error) {
+			return compressorFunc(src, width, height, depth, conv.dst)
+		})
+	}
+}
+
+func compressorFunc(src []byte, width int, height int, depth int, compressionFormat *image.Format) ([]byte, error) {
```

Comment:
compressorFunc -> compress
compressionFormat -> format or dstFormat

Reply:
You haven't addressed this.

Reply:
this is done